src/plugins/data/public/data_views/data_views/data_views_api_client.ts

```diff
@@ -51,27 +51,13 @@ export class DataViewsApiClient implements IDataViewsApiClient {
   }
 
   getFieldsForWildcard(options: GetFieldsOptions = {}) {
-    const { pattern, metaFields, type, params, dataSourceId } = options;
-
-    let url;
-    let query;
-
-    if (type) {
-      url = this._getUrl([type, '_fields_for_wildcard']);
-      query = {
-        pattern,
-        meta_fields: metaFields,
-        params: JSON.stringify(params),
-        data_source: dataSourceId,
-      };
-    } else {
-      url = this._getUrl(['_fields_for_wildcard']);
-      query = {
-        pattern,
-        meta_fields: metaFields,
-        data_source: dataSourceId,
-      };
-    }
+    const { pattern, metaFields, dataSourceId } = options;
+    const url = this._getUrl(['_fields_for_wildcard']);
+    const query = {
+      pattern,
+      meta_fields: metaFields,
+      data_source: dataSourceId,
+    };
```

Comment:
This chsnger caused `data_views/data_views_api_client.test.ts` unit tests to fail, so i had to update them at https://github.com/opensearch-project/OpenSearch-Dashboards/pull/10554 @kavilla can you verify that this is intended?

<img width="1308" height="662" alt="Screenshot 2025-09-25 at 10 07 03 AM" src="https://github.com/user-attachments/assets/ebc7b267-bbf9-49de-ad40-520fed7d0dab" />